SampleSize/SampleSizeCalculation.Rmd

```diff
@@ -0,0 +1,261 @@
+calc <- powCalc(psi, powFun)
+
+## choose one particular example at theta of 1 and power of 0.9
+pow <- powEx(calc, theta = 12, power = 0.9)
+
+## drawing the power plot with 3 contour lines
+plot(pow,
+     xlab = "Standard Deviation",
+     ylab = "Total Sample Size",
+     at = c(0.85, 0.9, 0.95))
+```
+
+For additional details, see the `sse` package documentation on [CRAN](https://CRAN.R-project.org/package=sse).
+
+### The precision based framework
+
+Suppose we want to estimate the proportion of deaths within 30 days of a myocardial infarction (MI). @law2002 estimated that 36% (31% - 40%) of patients died within 30 days of an MI. If we want to replicate their study, we can estimate the number of participants that would be necessary to achieve a CI which is 9% wide. Members of the SCTO Statistics and Methodology platform created an R package specifically for this problem called `presize` (@presize). It is available on [CRAN](https://CRAN.R-project.org/package=presize) or as an easy to use [shiny application](https://shiny.ctu.unibe.ch/presize/).
+
+The relevant function in presize is the `prec_prop` function. Plugging in the numbers from above, we can see that we require 434 participants (column `n`) to yield a confidence interval from 31.6% (`lwr`) to 40.6% (`upr`).
+
+```{r}
+library(presize)
+prec_prop(p = 0.36, conf.width = 0.09)
+```
+
+If we know that we can only afford to follow 300 participants, we can see what the confidence interval would be in that case too, via the `n` argument (instead of `cond.width`):
+
+```{r}
+prec_prop(p = 0.36, n = 300)
+```
+
+It's also possible to check multiple scenarios, by passing multiple values to a parameter. Below we vary the number of participants (what CI width is possible to determine with that number of participants?):
+```{r}
+prec_prop(p = .36, n = c(300, 350, 400, 450, 450))
+```
+
+The results can also be plotted easily, which is particularly useful when running multiple scenarios.
+
+```{r}
+library(ggplot2)  # for plotting
+library(magrittr) # for 'piping'
+
+prec_prop(p = .36, n = c(300, 350, 400, 450, 500)) %>% 
+  as.data.frame() %>% 
+  ggplot(aes(x = n, y = conf.width)) +
+  geom_line() +
+  labs(x = "N participants", 
+       y = "CI width")
+```
+
+For additional details and examples, see the [`presize` website](https://ctu-bern.github.io/presize/).
+
+# Conclusion
+
+Sample size calculation is a very important part of the study planning process. To make the most of your meeting with the statistician, have the answers to the questions mentioned above.
+
+# Quiz answers 
+
+Question 1: A (it's a false positive).  
+Question 2: B or C (a larger sample increases power which reduces the risk of a type 2 error).  
+Question 3: C (you're not testing a difference, just trying the estimate the quantity to within a given limit).  
+Question 4: B, C, and D (these are all important. E can also be useful for approximating trial duration.
```

Comment:
B? only the difference in means is needed, or am I missing something?

Reply:
you need an estimate of the variability too, so just the difference is not enough... or?

Reply:
yes, I meant C and D, but not B (I am referring to line 258)